### DISCUSSION TITLE
KeepKey Desktop Wallet Submission, V2 support

### DISCUSSION THREAD

**Discussion by BitHighlander** [WalletConnect/walletconnect-monorepo#2846] [Developer Support]
version 2.1.1 was released with full EIP:155 support for EVM's V2
 
Download here:
https://keepkey.com/get-started

Testing screenshots
# ping 
<img width="775" alt="Screen Shot 2023-06-27 at 7 41 47 PM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/4935895/5d93567c-3a9b-4aed-9e96-b6c8f53998ac">

Launcher:
```window.location.assign("keepkey://launch")``` launch as an electron intent

# pairing:
<img width="517" alt="Screen Shot 2023-06-27 at 7 10 16 PM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/4935895/400fbb34-791a-49bd-a649-56846462b985">
<img width="594" alt="Screen Shot 2023-06-28 at 10 28 38 AM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/4935895/ef6e4985-317a-43cb-8787-e6e210b0edc0">

# SignTx
<img width="566" alt="Screen Shot 2023-06-27 at 7 10 27 PM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/4935895/73dcf792-ff59-45a7-9494-c2a6d18fbd7e">

<img width="781" alt="Screen Shot 2023-06-27 at 7 32 39 PM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/4935895/31065bd4-2047-4a0f-8e40-f43049218b7c">

# sign 712 data

<img width="750" alt="Screen Shot 2023-06-27 at 8 02 43 PM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/4935895/673b5f36-7df5-452e-aaf1-fe18f606dd46">

v3 v4 (not in sample dapp) typed data supported and tested in the wild

# signMessage

<img width="775" alt="Screen Shot 2023-06-27 at 7 32 58 PM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/4935895/76b5dbee-67e3-4e8f-b9a8-7bd39ab96729">

<img width="775" alt="Screen Shot 2023-06-27 at 7 33 08 PM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/4935895/071cea19-168f-4fc3-96df-a67b69e8ee48">

**Comment by JeroldG** [comment]
HI Team, I have approved your submission.
